File: Practica 6/Ejercicio 2/suma_n_numeros.py
def suma_n_primeros_nums(num):
    cont = 0;
    for i in range (1, num +1):
        cont = cont + i
    print("\nLa suma de los",num,"primeros números es:",cont)
    
def main():
    num = (int(input("\nIntroduzca un número: ")))
    
    while (num < 0):
        input("\nError, introduzca un número pòsitivo")
        
    suma_n_primeros_nums(num)
    
if __name__ == "__main__":
    main()

File: Practicas/Practica 4/Ejercicio 2/suma_n_numeros.c
#include <stdio.h>

void suma_n_numeros_nums(int num) {
    int i;
    int suma = 0;
    for (i = 1; i <= num; i++) {
        suma += i;
    }

    printf("La suma de los primeros %d números enteros es: %d\n", num, suma);
    
}

int main() {

    int num;

    printf("Escriba un número: ");
    scanf("%d", &num);

    if (num >= 0) {
        suma_n_numeros_nums(num);
    } else {
        printf("Error, introduzca un número positivo");
    }

    return 0;
}